pack/return/rttcu_encounter.json

```diff
@@ -20,5 +539,569 @@
+        "traits": "Arkham.",
+        "type_code": "location"
+    },
+    {
+        "back_flavor": "If you didn’t know any better, you would think Southside was a ghost town. All of the residents have retreated into their homes and locked their doors. The only light you see seems to be coming from the Catholic church to the south.",
+        "clues": 0,
+        "code": "54052",
+        "double_sided": true,
+        "encounter_code": "return_to_in_the_clutches_of_chaos",
+        "encounter_position": 4,
+        "faction_code": "mythos",
+        "illustrator": "Cristi Balanescu",
+        "name": "Southside",
+        "pack_code": "rttcu",
+        "position": 52,
+        "quantity": 1,
+        "shroud": 1,
+        "text": "[action]: <b>Evade.</b> Use this ability on any enemy in play or in the encounter discard pile. If you fail, that enemy attacks you <i>(even if it is not in play)</i>. If you succeed, instead of evading the chosen enemy, move all breaches from Southside to the current act.",
+        "traits": "Arkham. Central.",
+        "type_code": "location"
+    },
+    {
+        "back_flavor": "Typically a safe neighborhood, the cobbled streets of Uptown have taken a very different mood this night. Tendrils of mist reach out from the woods that border the town, stretching over Hangman’s Brook and crawling through the streets.",
+        "clues": 0,
+        "code": "54053",
+        "double_sided": true,
+        "encounter_code": "return_to_in_the_clutches_of_chaos",
+        "encounter_position": 5,
+        "faction_code": "mythos",
+        "illustrator": "Cristi Balanescu",
+        "name": "Uptown",
+        "pack_code": "rttcu",
+        "position": 53,
+        "quantity": 1,
+        "shroud": 3,
+        "text": "[action]: Move an number of breaches from Uptown to the current act. Then, test [agility] (X), where X is the number of breaches moved. For each point you fail by, take 1 damage.",
+        "traits": "Arkham.",
+        "type_code": "location"
+    },
+    {
+        "back_flavor": "The massive Romanesque building towers over the rest of Uptown, light shining from within its stained glass windows. Father Michael has made it a point to keep the church open at all hours, and despite the strange happenings throughout town, tonight is no different.",
+        "clues": 0,
+        "code": "54054",
+        "double_sided": true,
+        "encounter_code": "return_to_in_the_clutches_of_chaos",
+        "encounter_position": 6,
+        "faction_code": "mythos",
+        "flavor": "Many fearful citizens are seated at the church's wooden pews, praying and whispering amongst themselves.",
+        "illustrator": "Mark Molnar",
+        "name": "South Church",
+        "pack_code": "rttcu",
+        "position": 54,
+        "quantity": 1,
+        "shroud": 1,
+        "text": "[action][action]: Move all breaches from South Church to the current act.\n[action]: <b>Resign.</b> You hide through the night.",
```

Comment:
Please add a space between the double actions, as per the database:

`[action] [action]`